services/app-api/prisma/schema.prisma

```diff
@@ -64,7 +79,39 @@ model ContractRevisionTable {
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now()) @updatedAt
 
-  name String
+  submissionType                               SubmissionType
+  submissionDescription                        String?
+  programIDs                                   String[]
+  populationCovered                            PopulationCoverageType?
+  riskBasedContract                            Boolean?
+  stateContacts                                StateContact[]
+  addtlActuaryContacts                         ActuaryContact[]
+  addtlActuaryCommunicationPreference          ActuaryCommunication?
+  supportingDocuments                          ContractSupportingDocument[]
+  contractType                                 ContractType?
+  contractExecutionStatus                      ContractExecutionStatus?
+  contractDocuments                            ContractDocument[]
+  contractDateStart                            DateTime?
```

Comment:
All of these dates that are entered by users are CalendarDates not DateTimes. This is represented in postgres with the `@db.Date` see https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#postgresql-6

we should be using that everywhere that isn't a date time